### PR TITLE
Request service ui

### PR DESCRIPTION
**Pull request by deepali1-aconnexion**

### CODE REVIEW

File: app/views/locations/service_requests/_form.html.slim [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          h4.text-center When would you like a technician to arrive?
+          = f.input :work_time_details, as: :text, label: false
+          h4 When the technician arrives, who should they ask for?
+          = f.input :full_name, label: false, input_html: { placeholder: 'Full Name' }
+          = f.input :email, label: false, input_html: { placeholder: 'Email' }
+          = f.input :phone_number, label: false, input_html: { placeholder: 'Phone Number' }
+          = f.input :company_name, label: false, input_html: { placeholder: 'Company Name' }
+          .clearfix
+          .col-md-9.margintop30.center-panel
+            = f.submit 'SCHEDULE', class: 'btn btn-continue btn-schedule-service', data: { payment: "#{current_customer.current_account.have_payment_method}"}
```
Comment (AlexanderUlitin) [inline app/views/locations/service_requests/_form.html.slim:66]: Please uppercase by CSS.

File: app/views/locations/service_requests/_form.html.slim [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            strong Your credit card will never be charged without your approval
+          .line-spacer
+          .icon-lock
+            i.fa.fa-lock aria-hidden="true" 
+          div
+            = image_tag 'powerdBy.png', height: '35', width: '125'
+          h3 Sendaguy now uses Stripe, a leader in online and mobile payments, for all financial transactions.
+          .row
+            .col-sm-12
+              = f.submit 'SCHEDULE', class: 'btn btn-continue'
```
Comment (AlexanderUlitin) [inline app/views/locations/service_requests/_form.html.slim:89]: Please uppercase by CSS.

File: app/views/locations/service_requests/new.html.slim [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.service-request-form-wrapper
+  = render 'form', service_request: @service_request, location: @location, categories: @categories
+                  
+                      
+                
+                
+                
+
```
Comment (AlexanderUlitin) [inline app/views/locations/service_requests/new.html.slim:8]: Too many empty lines, can we remove this?

File: config/initializers/secure_headers.rb [hunk 4]
```diff
@@ -9,8 +9,7 @@
   config.csp.merge!(
     default_src: %W(#{ENV['APP_PROTOCOL']}: 'self'),
     img_src: %w('self' https://maps.googleapis.com https://csi.gstatic.com https://*.s3.amazonaws.com data: https://maps.gstatic.com),
-    script_src: %w('self' https://maps.googleapis.com 'unsafe-eval'),
-    script_src: %w('self' https://js.stripe.com/v2/ 'unsafe-eval'),
+    script_src: %w('self' https://maps.googleapis.com https://js.stripe.com/v2/ ),
```
Comment (AlexanderUlitin) [inline config/initializers/secure_headers.rb:12]: Please double-check and make sure that all JS functionality works fine without `'unsafe-eval'` allowed here?

**Comment by deepali1-aconnexion** [comment]
Bugs fixed.

**Pull request merged**